### PR TITLE
Active directory query v2 a santamarina

### PR DESCRIPTION
**Pull request by asantamarina**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A 

## Description
Modifies the AD account attribute "Password never expire".  

## Screenshots
N/A

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py [hunk 1]
```diff
@@ -1637,6 +1637,58 @@ def get_mapping_fields_command(search_base):
...
+
+    # Query by sAMAccountName
+    sam_account_name = escape_filter_chars(sam_account_name)
+    query = "(&(objectClass=User)(objectCategory=person)(sAMAccountName={}))".format(sam_account_name)
+    entries = search_with_paging(query, default_base_dn, attributes='userAccountControl')
+
+    if not check_if_user_exists_by_attribute(default_base_dn, "sAMAccountName", sam_account_name):
+        raise Exception(f"sAMAccountName {sam_account_name} was not found.")
+
+    # TODO: make sure that if exist - its a list with at least one element and user_account_control always exist.
```
Comment (ShacharKidor) [inline Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py:1656]: need to verify that.

File: Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md [hunk 2]
```diff
@@ -92,7 +92,26 @@ Expires the password of an Active Directory user.
...
+##### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| username | The username (samAccountName) of the user to modify. | Required |
+| value | Value to set "Password Never Expire" (default is false). | Required |
+
+##### Context Output
+
+There is no context output for this command.
```
Comment (ShacharKidor) [inline Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md:112]: There aren't any outputs for that command?

**Comment by ShacharKidor** [comment]
Hi @jrosenpanw - the only thing left here is the documentation review.
Can you please assign one of your team members? 

Thank you :)

**Comment by ShirleyDenkberg** [comment]
@ShacharKidor Doc review completed.

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**